collect_app/src/main/java/org/odk/collect/android/utilities/EncryptionUtils.java

```diff
@@ -154,8 +154,7 @@ private EncryptionUtils() {
                 byte[] pkEncryptedKey = pkCipher.doFinal(key);
                 String alg = pkCipher.getAlgorithm();
                 Timber.i("AlgorithmUsed: %s", alg);
-                base64RsaEncryptedSymmetricKey = wrapper
-                        .encodeToString(pkEncryptedKey);
+                base64RsaEncryptedSymmetricKey = Base64.encodeToString(pkEncryptedKey, FLAGS);
```

Comment:
While we’re here let’s change `AlgorithmUsed` to `Algorithm used`.

Reply:
Done